src/InProcessTestHost/InProcessTestHost.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Microsoft.DurableTask.Testing</RootNamespace>
+    <AssemblyName>Microsoft.DurableTask.InProcessTestHost</AssemblyName>
+    <PackageId>Microsoft.DurableTask.InProcessTestHost</PackageId>
+    <EnableStyleCop>false</EnableStyleCop>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
What is this warning that we're supressing? Can we add a comment here to explain?